src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -205,11 +214,21 @@ private void setParameterMap(
         parameterOverrides,
         props);
 
+    this.updateValue(
+        DISABLE_CHUNK_ENCRYPTION,
+        isIcebergMode
+            ? DISABLE_CHUNK_ENCRYPTION_ICEBERG_MODE_DEFAULT
+            : DISABLE_CHUNK_ENCRYPTION_DEFAULT,
+        parameterOverrides,
```

Comment:
this allows SDK customers to specify whether chunk encryption is enabled or disabled, why do we want to do that?